server/models/User.js

```diff
@@ -0,0 +1,28 @@
+module.exports = (sequelize, DataTypes) => {
+    const User = sequelize.define('User', {
```

Comment:
Looks good. We will probably have to add different roles in the future for Rutwa to coordinate students. But dw about that for now.

Reply:
Since we are pivoting to use uoft credentials for auth (https://isea.utoronto.ca/services/weblogin/sso-howto-weblogin/), wondering if we need to change the model to handle that but yea can worry about that later